spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -802,3 +802,120 @@ definitions:
+
+ - OrbitClockBound:
+     desc: Orbit and clock bound.
+     fields:
+       - sat_id:
+           type: u8
+           desc: >
+             Satellite ID. Encoded following either RTCM DF068 (GPS), DF252 (Galileo),
+             or DF488 (BDS) according to the constellation.
+       - orb_radial_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_radial_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Mean (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Standard Deviation (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+
+ - MSG_SSR_ORBIT_CLOCK_BOUNDS:
+     id: 0x05DE
+     short_desc: Combined Orbit and Clock Bound
+     public: False
+     fields:
+       - header:
+           type: BoundsHeader
+           desc: Header of a bounds message.
+       - ssr_iod:
+           type: u8
+           desc: IOD of the SSR bound. Encoded following RTCM DF413 specification.
+       - const_id:
+           type: u8
+           desc: Constellation ID to which the SVs belong.
+       - nb_sat:
+           type: u8
+           desc: Number of satellites. Encoded following RTCM DF387 specification.
```

Comment:
> Encoded following RTCM DF387 specification.

Is this true for this particular field?

Reply:
Maybe you are right. In the Excel file we are saying that this field is similar to the existing RTCM Data Field. Perhaps `Encoded` isn't the best choice of word. (And also DF387 is only 6 bits long)

Reply:
Maybe "Number of satellites, follows the RTCM DF387 specification." -- if it's only expected to be 6-bit, we should also document the valid ranges (0-63).

Reply:
Yes probably we should drop the reference to RTCM here. After all it's just a number?
And I think we won't limit the range to only 6-bit.

Reply:
I think if we design these messages to more closely match their RTCM equivalents, we'll have an easier time when we go through the conversion effort.  Data fidelity and correctness is more important than user experience.  So... the question is, do we want do this now... or later?

Risks:
- On doing it now: delaying progress on getting the end-to-end solution working with SBP
- On doing it later: it'll get forgotten or we'll be in a situation where we'll lose fidelity or have to drop data when converting to RTCM

In general this also relates to this issue here: https://github.com/swift-nav/libsbp/pull/1108#discussion_r850505904

Reply:
These messages don't have RTCM equivalents. Any conversion to RTCM will be just wrapping them in the Swift proprietary RTCM message. The fields which are being referenced from RTCM though exist in other RTCM SSR message types which we will need to compare the values against, so there it makes sense to use a similar definition.

Reply:
Alright, @woodfell's suggestion makes sense in that case

Reply:
Removed all the references to RTCM where the field is just a number. Left the ones regarding satellites IDs and update interval since they have some special meaning. The comment now reads `Similar to RTCM DF` for context.